torchaudio/datasets/librispeech.py

```diff
@@ -23,22 +34,32 @@
 }
 
 
+def download_librispeech(root, url):
+    ext_archive = ".tar.gz"
+    filename = url + ext_archive
+    archive = os.path.join(root, filename)
+    base_url = "http://www.openslr.org/resources/12/"
+    download_url = os.path.join(base_url, filename)
+    if not os.path.isfile(archive):
+        checksum = _CHECKSUMS.get(download_url, None)
+        download_url_to_file(download_url, archive, hash_prefix=checksum)
+    extract_archive(archive)
+
+
 def load_librispeech_item(
-    fileid: str, path: str, ext_audio: str, ext_txt: str
```

Comment:
FYI: the reason why all the dataset load functions have this `ext_audio` argument is for testing. In unit test we generate mock data and we use WAV format, for the simplicity. (because there aren't many easy-to-use tool to generate non-WAV audio) To test the functionality of this function, we need overwrite of extension, so we have this extra argument.

see 
https://github.com/pytorch/audio/blob/595dc5d3a3ebf19a5a6d0f538bf2717f30f06b1a/test/torchaudio_unittest/datasets/librispeech_test.py#L88

Reply:
got it, thanks